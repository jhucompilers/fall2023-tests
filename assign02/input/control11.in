var a;
var b;
var c;
var d;

a = 3;

b = 99;
c = 101;
d = 103;

if (a) {
  var b;
  b = 65;
  if (a) {
    var c;
    c = 66;
    if (a) {
      d = 67;
    }
  }
}

println(b);
println(c);
println(d);
a;
