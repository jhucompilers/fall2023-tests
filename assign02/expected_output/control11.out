99
101
67
Result: 3
